sound/soc/intel/boards/sof_pcm512x.c

```diff
@@ -136,6 +139,9 @@ static int sof_card_late_probe(struct snd_soc_card *card)
 	if (list_empty(&ctx->hdmi_pcm_list))
 		return -EINVAL;
 
+	if (!ctx->idisp_codec)
```

Comment:
unrelated to this patch: the above "return -EINVAL;" can only be triggered if for some reason .init() wasn't called for any of the three iDisp interfaces. A comment would be nice

Reply:
@lyakh yeah, this is in all drivers and has been carried over. I think this would be triggered by an invalid topology file, but not 100% sure... would need some investigation to write a good comment.